.github/workflows/autolabel-pullrequests.yml

```diff
@@ -0,0 +1,122 @@
+              case /^orc8r\/tools\/packer\/.*/.test(f.filename):
+              case /^orc8r\/cloud\/deploy\/bare-metal\/.*/.test(f.filename):
+              case /^orc8r\/cloud\/deploy\/bare-metal-ansible\/.*/.test(f.filename):
+              case /^lte\/gateway\/docker\/.*/.test(f.filename):
+              case /^lte\/gateway\/release\/.*/.test(f.filename):
+              case /^lte\/gateway\/Vagrantfile/.test(f.filename):
+              case /^cwf\/gateway\/Vagrantfile/.test(f.filename):
+                console.log("file changed under CI component : " + f['fileame']);
+                lbls.push("component: ci");  
+                break;
+              case /^nms\/.*/.test(f.filename):
+                console.log("file changed under NMS component : " + f['fileame']);
+                lbls.push("component: nms");  
+                break;
+              case /^cwf\/.*/.test(f.filename):
+              case /^feg\/radius\/.*/.test(f.filename):
+                console.log("file changed under CWF component : " + f['fileame']);
+                lbls.push("component: cwf");  
+                break;
+              case /^feg\/.*/.test(f.filename):
+                console.log("file changed under FEG component : " + f['fileame']);
+                lbls.push("component: feg");  
+                break;
+              case /^openwrt\/.*/.test(f.filename):
+                console.log("file changed under OPENWRT component : " + f['fileame']);
+                lbls.push("component: openwrt");  
+                break;
+              case /^lte\/gateway\/.*/.test(f.filename):
+              case /^lte\/protos\/.*/.test(f.filename):
+              case /^orc8r\/gateway\/c\/.*/.test(f.filename):
+              case /^third_party\/gtp_ovs\/.*/.test(f.filename):
+                console.log("file changed under AGW component : " + f['fileame']);
+                lbls.push("component: agw");  
+                break;
+              case /^\.golangci\.yml/.test(f.filename):
+              case /^orc8r\/.*/.test(f.filename):
+                console.log("file changed under ORC8R component : " + f['fileame']);
+                lbls.push("component: orc8r");  
+                break;
+              case /^xwf\/.*/.test(f.filename):
+                console.log("file changed under XWF component : " + f['fileame']);
+                lbls.push("component: xwf");  
+                break;
+              case /^show-tech\/.*/.test(f.filename):
+                console.log("file changed under SHOW-TECH component : " + f['fileame']);
+                lbls.push("component: show-tech");  
+                break;
+              case /^docs\/.*/.test(f.filename):
+                console.log("file changed under DOCS: " + f['fileame']);
+                lbls.push("component: docs");  
+                break;
+             } // end of switch case
+            } // end of for loop
+
+            let uniqLbls = new Set(lbls);
```

Comment:
you could have created a set from the beginning.

Reply:
@quentinDERORY Eventually I still have to convert the Set in to an array as the API call takes only the array of strings for the labels.